dash_bio/component_factory/_clustergram.py

```diff
@@ -45,131 +45,120 @@ def Clustergram(
-- color_threshold (dict; optional): Maximum linkage value for which
-    unique colors are assigned to clusters; 'row' for rows, and 'col'
-    for columns. (Default: {'row': 0, 'col': 0})
-- optimal_leaf_order (bool; optional): Enabling/disabling of the
-    option to determine leaf order that maximizes similarity between
-    neighboring leaves. (Default: False)
-- color_map (list; optional): The colorscale for the heatmap. Each
-    list element contains two elements; the first element refers to
-    the portion of the maximum data point under which a cell will be
-    colored, and the second element refers to the color. e.g., a
-    colorscale [[0.0, 'white'], [0.5, 'gray'], [1.0, 'black']] mean
-    that for all cells with a value in the 50th or lower percentile of
-    the dataset, the color on the heatmap would be white; all cells
-    with a value in the 50th or higher percentile, excluding the 100th
-    percentile, would be gray; and the cell(s) in the 100th percentile
-    would be colored black. (Default: [[0.0, 'rgb(255,0,0)'], [0.5,
-    'rgb(0,0,0)'], [1.0, 'rgb(0,255,0)']])
+    column-clustered data.
+- row_dist (string; default 'euclidean'): Distance metric for rows.
+    Passed as argument `metric` to the function specified in `dist_fun`.
+- col_dist (string; default 'euclidean'): Distance metric for columns.
+    Passed as argument `metric` to the function specified in `dist_fun`.
+- dist_fun (function; default scipy.spatial.distance.pdist): Function
+    to compute the pairwise distance from the observations (see docs for
+    scipy.spatial.distance.pdist).
+- link_fun (function; default scipy.cluster.hierarchy.linkage): Function to
+    compute the linkage matrix from the pairwise distances (see docs for
+    scipy.cluster.hierarchy.linkage).
+- color_threshold (dict; default {'row': 0, 'col': 0}): Maximum
+    linkage value for which unique colors are assigned to clusters;
+    'row' for rows, and 'col' for columns.
+- optimal_leaf_order (bool; default False): Whether to enable (True) or
+    disable (False) the option to determine leaf order that maximizes
+    similarity between neighboring leaves.
+- color_map (list; default [[0.0, 'rgb(255,0,0)'], [0.5,
+    'rgb(0,0,0)'], [1.0, 'rgb(0,255,0)']]): Colorscale for the heatmap.
+    Top-level elements contain two elements, the first of which refers to
+    the percentile rank, and the second to the applied color. For instance,
+    [[0.0, 'white'], [0.5, 'gray'], [1.0, 'black']] means that cells in the
+    49th percentile would be white; cells at the 50th or higher percentiles,
+    excluding the 100th percentile, would be gray; and the cell(s) at the
+    100th percentile would be black.
 - color_list (dict; optional): The list of colors to use for different
    clusters in the dendrogram that have a root under the threshold for
    each dimension. If there are fewer colors than there are clusters
    along a specific dimension. The keys are: 'row' (for row clusters),
```

Comment:
"If there are ..." ? There's a syntax issue here (in natural language).

Reply:
Fixed in b7a6c49.